Source/WebInspectorUI/UserInterface/Views/SpreadsheetCSSStyleDeclarationEditor.js

```diff
@@ -60,7 +61,13 @@ WI.SpreadsheetCSSStyleDeclarationEditor = class SpreadsheetCSSStyleDeclarationEd
         if (!this._style)
             return;
 
-        this.element.addEventListener("focus", () => { this.focused = true; }, true);
+        this.element.addEventListener("focus", () => {
+            if (this._selectionFocusOverride)
+                this._selectionFocusOverride = false;
```

Comment:
can you explain more why this is needed?  is it that `this.editing` is now `true`?  perhaps the issue is with `this._style.locked`?

Reply:
The `layout()` function of this class terminates when `editing` is true. Until this commit, `_focused` was always set to true, as usually focusing meanings that we are editing the CSS and we shouldn't layout. However focusing is _also_ done when an item is highlighted (which AFAIK is either when it is arrived at from a go to arrow or other UI element, or when it is selected manually by the user by clicking in the row but outside the text field). 

So when the go to arrow was clicked, it would prevent the initial layout and therefore all of the properties could appear blank. This didn't occur with two-column view since the styles pane was already layouted.

`this._style.locked` doesn't seem to be checked as part of this initial condition in the `layout()` function, so I don't think that's the cause.

Reply:
Yeah, I did think the same thing - that ideally focus wouldn't do double-duty for two disparate things, but it's probably not something to do rn. I also think there may be AX concerns with unbundling it, given that both of them are "focusing" in an accessibility sense.

I have already tried `this.didInitialLayout` but it's not false when necessary (since AFAIK it's changed after `initialLayout` but before the first `layout` in `View`). Similarly `isTrusted` is always true.